wgsl/index.bs

```diff
@@ -5126,14 +5130,14 @@ textureLoad(t : texture_depth_2d, coords : vec2<i32>) -> f32
 textureLoad(t : texture_depth_2d, coords : vec2<i32>, level : i32) -> f32
 textureLoad(t : texture_depth_2d_array, coords : vec2<i32>, array_index : i32) -> f32
 textureLoad(t : texture_depth_2d_array, coords : vec2<i32>, array_index : i32, level : i32) -> f32
-textureLoad(t : [[access(read)]] texture_storage_1d<F>, coords : i32) -> vec4<T>
-textureLoad(t : [[access(read)]] texture_storage_1d_array<F>, coords : i32, array_index : i32) -> vec4<T>
-textureLoad(t : [[access(read)]] texture_storage_2d<F>, coords : vec2<i32>) -> vec4<T>
-textureLoad(t : [[access(read)]] texture_storage_2d_array<F>, coords : vec2<i32>, array_index : i32) -> vec4<T>
-textureLoad(t : [[access(read)]] texture_storage_3d<F>, coords : vec3<i32>) -> vec4<T>
+textureLoad(t : texture_storage_1d<F>, coords : i32) -> vec4<T>
```

Comment:
There is no indication I can't pass a write only storage texture here anymore.

Reply:
Yes, but this isn't a legal signature, right? I did update the parameter description below.
I want to remove these as intrinsics like `textureDimensions` will not include access annotations.